app/src/main/res/drawable/shape_shadow_of_game_card.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Where is shadow in **Game Screen ( snafer )**?

Reply:
the file has been added by the wrong and I forgot to delete it. 
Done